<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>294</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tabGeneral">
        <attribute name="title">
         <string>General</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Destination</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLineEdit" name="lineEditDestination">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="buttonOpenDestination">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Open</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Ending Format:</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxFormat">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxNumberPrefix">
             <property name="text">
              <string>Add number at the beginning of a file name.</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QSpinBox" name="spinBoxMaxDownloads">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Number of simultaneous downloads.</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Amazon</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>The amazon web page the actions should relate to:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>http://www.Amazon.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxAmazonUrl">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonOk">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>&amp;Ok</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonApply">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>&amp;Apply</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonCancel">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>C&amp;ancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonApply</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxFormat</sender>
   <signal>activated(QString)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>updateButtons()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOpenDestination</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>openDestination()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditDestination</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>updateButtons()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxAmazonUrl</sender>
   <signal>activated(QString)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>updateButtons()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openDestination()</slot>
  <slot>updateButtons()</slot>
  <slot>apply()</slot>
  <slot>ok()</slot>
 </slots>
</ui>
